test/with_dummyserver/test_socketlevel.py

```diff
@@ -1930,7 +1932,7 @@ def socket_handler(listener: socket.socket) -> None:
         from urllib3.connectionpool import log
 
         with mock.patch.object(log, "warning") as log_warning:
-            with HTTPConnectionPool(self.host, self.port, timeout=3) as pool:
+            with HTTPConnectionPool(self.host, self.port, timeout=0.5) as pool:
```

Comment:
We have multiple tests with timeout=3 here, but we should probably be using LONG_TIMEOUT instead, which is longer in CI.